Makefile

```diff
@@ -345,6 +348,12 @@ SIM_NUM_BLOCKS ?= 500
 SIM_BLOCK_SIZE ?= 200
 SIM_COMMIT ?= true
 
+#? test-sim-fuzz: Run fuzz test for simapp
+test-sim-fuzz:
+	@echo "Running application fuzz for numBlocks=2, blockSize=20. This may take awhile!"
+#ld flags are a quick fix to make it work on current osx
+	@cd ${CURRENT_DIR}/simapp && go test -mod=readonly -json -tags='sims' -ldflags="-extldflags=-Wl,-ld_classic" -timeout=60m -fuzztime=60m -run=^$$ -fuzz=FuzzFullAppSimulation -GenesisTime=1714720615 -NumBlocks=2 -BlockSize=20
```

Comment:
@ggrieco-tob this target contains a 1h timeout. It is probably not related to the issues that you were facing but I want to highlight this setting, just in case. 
I used 1h for fuzz tests on CI but this can be configured somewhere else, too. Please let me know if other default timeout values make sense.

Reply:
I'm testing removing the 1 hour timeout parameters but still fails for longer campaign. Either the timeout is enforced anyway or there is another issue.